### PR TITLE
[QA-2108] feat: add matrix in e2e tests

### PR DESCRIPTION
**Pull request by rafael-qazion**
### What is the new behavior introduced by this PR?
Matrix in workflow (parallel tests)

### Does this PR introduce breaking changes?
- [X] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
No

### Does it have a link on Figma?
No

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [X] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [X] Commits are tagged with the right word (feat, test, refactor, etc)
- [X] Application responsiveness was tested to different screen sizes
- [X] Code is formatted and linted
- [X] Tags are added to the PR

#### These changes were tested on the following browsers:
- [X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: cypress/e2e/domains/create-domain-lets-encrypt.cy.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const createEdgeApplicationCase = () => {
   cy.get(selectors.domains.pageTitle(edgeAppName)).should('have.text', edgeAppName)
 }
 
-describe('Domains spec', () => {
+describe('Domains spec', { tags: ['@dev3', 'xfail'] }, () => {
```
Comment (matheus-sunderhus) [inline cypress/e2e/domains/create-domain-lets-encrypt.cy.js:23]: Acredito que faltou um @ no xfail

**Review by matheus-sunderhus — COMMENTED** [review]
minor changes

File: sonar-project.properties [hunk 2]
```diff
@@ -12,6 +12,5 @@ sonar.javascript.lcov.reportPaths=lcov.info
 sonar.sources=.
 sonar.tests=.
 sonar.test.inclusions=**/*.test.js
-sonar.exclusions=**/*.test.js, **/*.spec.js
-sonar.coverage.exclusions=src/utils/**, src/store/**, src/services/**, src/plugins/**,  __ENV__/**
-
+sonar.exclusions=**/*.test.js, **/*.spec.js, cypress/**
+sonar.coverage.exclusions=src/utils/**, src/store/**, src/services/**, src/plugins/**,  cypress/**, __ENV__/**
```
Comment (alisson-marcelo-azion) [inline sonar-project.properties:16]: cypress/** nessa linha provavelmente não é necessário, pois já está na lista de exclusões de análise.

File: sonar-project.properties [hunk 3]
```diff
@@ -12,6 +12,5 @@ sonar.javascript.lcov.reportPaths=lcov.info
 sonar.sources=.
 sonar.tests=.
 sonar.test.inclusions=**/*.test.js
-sonar.exclusions=**/*.test.js, **/*.spec.js
-sonar.coverage.exclusions=src/utils/**, src/store/**, src/services/**, src/plugins/**,  __ENV__/**
-
+sonar.exclusions=**/*.test.js, **/*.spec.js, cypress/**
```
Comment (alisson-marcelo-azion) [inline sonar-project.properties:15]: Não sei se temos algum arquivo spec.js no projeto, mas isso já estava assim no arquivo.

**Review by alisson-marcelo-azion — APPROVED** [review]

**Review by matheus-sunderhus — APPROVED** [review]

**Pull request merged**